bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TypedescValue.java

```diff
@@ -26,5 +26,10 @@
  * @since 1.3.0
  */
 public interface TypedescValue extends RefValue, BTypedesc {
-
+    /**
+     * Returns the annotations specified on this type descriptor.
+     *
+     * @return annotations
+     */
+    MapValue getAnnotations();
```

Comment:
Related to what we discussed offline. Shall we avoid adding this method for now? Ideally eventually we should have this (only) in the typedesc.